templates/toil-instance.yaml

```diff
@@ -72,6 +72,29 @@ Parameters:
   PublicRsaKey:
     Description: Public RSA key for instance
     Type: String
+  ToilAppliance:
+    Description: Docker image to use for the toil instances.
+    Type: String
+    Default: 'quay.io/tessthyer/amp-rnaseq_reprocess:toil-3.20.0'
+  ClusterName:
+    Description: Name of the cluster, will be applied to all instances associated with it.
+    Type: String
+  ClusterLeaderNodeType:
+    Description: EC2 instance type for the leader node in the toil cluster.
+    Type: String
+    Default: 'm5.large'
```

Comment:
would it make sense to constrain this to a small(ish) set of instance types?  would a compute or memory optimized instance be better?

Reply:
Probably, since this is for the leader, which is mostly orchestrating, not the worker nodes, which will be doing the actual heavy lifting. Whether the workers will need to be more compute-intensive or memory-intensive depends on the type of work they're doing. I might be comfortable restricting the leader to general purpose class nodes T3 and M5. Do you have suggestions on things you would like to see restricted? In any case I'd like to leave this restriction for another refactor as we are still testing how the clusters perform. I think it's premature to restrict the type at this point.